src/collective/blueline/browser/viewlets.py

```diff
@@ -31,6 +34,25 @@ def show(self):
         show = api.user.is_anonymous() or self.show_authenticated
         return not is_configlet and show
 
+    def evaluate_code(self, code):
```

Comment:
@rodfersou can we better use here the [Chameleon compiler](https://chameleon.readthedocs.org/en/latest/library.html?highlight=compile#module-chameleon.compiler)?